README.md

```diff
@@ -206,6 +206,10 @@ View [the library](https://github.com/yapplabs/ember-key-responder) for more inf
 * `ember test`
 * `ember test --server`
 
+## Unit Tests
+
+When running unit tests on components that use ember-modal-dialog it is necessary to create and register the container for  ember-modal-dialog to wormhole itself into.  See this [example](https://github.com/yapplabs/ember-modal-dialog/tree/master/tests/unit/components/component-that-uses-modal-dialog-test.js) for how to set this up in a unit test.
```

Comment:
It would be nice to use a [relative link](https://help.github.com/articles/relative-links-in-readmes/) here that will work on all branches. 

Looks really good overall.. thanks!